### PR TITLE
Update SODA.tex

### PR DESCRIPTION
**Pull request by Bonnarel**
RElated to issue #4 this Commit introduces a couple of subsections for new rebinning / reprojecting parameters in SODA

### CODE REVIEW

**Comment by Bonnarel** [comment]
Related to issue #4 this PR introduces a couple of subsections for new rebinning / reprojecting parameters in SODA

**Review by jd-au — APPROVED** [review]

File: SODA.tex [hunk 1]
```diff
@@ -654,6 +654,47 @@ \subsubsection{POL}
 The UCD  describing the POL parameter is
 \ucd{meta.code;phys.polarization}.
 
+\subsubsection{SPATRES}
```
Comment (jd-au) [inline SODA.tex:670]: These are currently under the filtering parameters subsection (3.3) Should they instead be under a new subsection such as transformation parameters ?

**Review by jd-au — APPROVED** [review]

**Comment by Bonnarel** [comment]
Le 26/04/2022 à 10:41, James Dempsey a écrit :
>
> @jd-au commented on this pull request.
>
> In [SODA.tex](https://github.com/ivoa-std/SODA/pull/12#discussion_r858447409):
> > @@ -654,6 +654,47 @@ \subsubsection{POL}
>  The UCD  describing the POL parameter is
>  \ucd{meta.code;phys.polarization}.
>  
> +\subsubsection{SPATRES}
>
> These are currently under the filtering parameters subsection (3.3) Should they instead be under a new subsection such as transformation parameters ?

I wrongly marked this as resolved. This is a very good idea. I'll change that.

**Comment by Bonnarel** [comment]
Le 26/04/2022 à 10:41, James Dempsey a écrit :
>
> ***@***.**** commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In SODA.tex 
> <https://github.com/ivoa-std/SODA/pull/12#discussion_r858447409>:
>
> > @@ -654,6 +654,47 @@ \subsubsection{POL}
>   The UCD  describing the POL parameter is
>   \ucd{meta.code;phys.polarization}.
>   
> +\subsubsection{SPATRES}
>
> These are currently under the filtering parameters subsection (3.3) 
> Should they instead be under a new subsection such as transformation 
> parameters ?
>
I wrongly marked this as resolved. This is a very good idea. I'll change 
that.

**Pull request closed without merging**